### PR TITLE
Fix member name on optimistic message

### PR DESCRIPTION
**Pull request by zia-r**
Use the member's name for the optimistic message update on send. 

Also add on a class for redacted messages allowing consumers to style redacted messages. We do not have built-in default styling for redacted messages.

### CODE REVIEW

File: src/components/SourceElements.tsx [hunk 1]
```diff
@@ -16,11 +16,20 @@ export interface SourceElementsProps {
 }
 
 export const SourceElements: FunctionComponent<SourceElementsProps> = ({ client, children }) => {
+  const [member, setMember] = useState<Member | null>(null)
+  useEffect(() => {
+    client.members.retrieve('current').then((member) => {
+      console.log('Member retirved', member)
```
Comment (colinmorelli) [inline src/components/SourceElements.tsx:22]: Let's not console log here

File: src/components/SourceElements.tsx [hunk 2]
```diff
@@ -16,11 +16,20 @@ export interface SourceElementsProps {
...
+      setMember(member)
+    })
+  }, [])
```
[the remark this answers is not in the document]
Comment (colinmorelli) [reply src/components/SourceElements.tsx]: This actually won't be enough. You can call `client.configuration.setAuthentication` on the client directly, to change auth. That wouldn't be caught here, and you wouldn't see the updated member.

Is this maybe where it makes sense to provide an authentication instance directly to Elements and then we'll create the client on your behalf?

[the remark this answers is not in the document]
Comment (zia-r) [reply src/components/SourceElements.tsx]: yeah that would work - it's a "breaking" change in terms of an update to `SourceElement` props

[the remark this answers is not in the document]
Comment (colinmorelli) [reply src/components/SourceElements.tsx]: We could keep the client option for backwards compat, but flag this difference and recommend passing an authentication instance?

**Pull request merged**